### PR TITLE
Add Dutch translation

### PR DESCRIPTION
**Pull request by M-BTC**

### CODE REVIEW

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+  <string name="receive_address_label">Adres</string>
+  <string name="receive_power_save_disclaimer">Je gebruikt de energie-stand. Om de betaling te ontvangen heeft de app toestemming nodig om op de achtergrond te draaien.</string>
+  <string name="receive_withdraw_button">Gebruik LNURL link</string>
+
+  <string name="receive_swap_in_button">Laat Bitcoinadres zien</string>
+  <string name="receive_swap_in_error_message">Er is een fout opgetreden tijdens het wisselen naar een on-chain transactie…\n\nProbeer opnieuw.</string>
+  <string name="receive_swap_in_progress_message">Swap-in Bitcoinadres aanvragen…</string>
+  <string name="receive_swap_in_disclaimer_title">Ontvang on-chain</string>
+  <string name="receive_swap_in_disclaimer_message"><![CDATA[
+  A standard Bitcoin address will be displayed next.
```
Comment (bitcoinuser) [inline app/src/main/res/values-nl/strings.xml:100]: Needs to be translated

File: app/src/main/res/values-nl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+  <br /><br />
+  Funds sent to this address will be shown on your wallet after one confirmation.
```
Comment (bitcoinuser) [inline app/src/main/res/values-nl/strings.xml:102]: Needs to be translated

File: app/src/main/res/values-nl/strings.xml [hunk 3]
```diff
@@ -0,0 +1,468 @@
...
+  <br /><br />
+  A <b>%1$s%%</b> fee will be applied.
```
Comment (bitcoinuser) [inline app/src/main/res/values-nl/strings.xml:104]: Needs to be translated

**Comment by M-BTC** [comment]
> Needs to be translated.

Sorry, missed those. They've been translated!

**Comment by dpad85** [comment]
Great! Let me know when you think this is ready for review.

**Comment by M-BTC** [comment]
Will do! I'll share it in a few Telegram groups for additional checking first.

**Comment by M-BTC** [comment]
Hey! I've had a few other Dutch people look at the translation and made some minor changes. We feel like it's good for your final review!

**Pull request merged**